xml/quick_configuration.xml

```diff
@@ -132,14 +127,26 @@
      <term>Host</term>
       <listitem>
        <para>
+        Hostname of the LDAP server used for this connection.
+       </para>
+       <note>
+        <title>Provide the hostname as FQDN</title>
+       <para>
+        The <guimenu>Host</guimenu> field must use a Fully 
+        Qualified Domain Name, as IP address is not allowed with TLS.
        </para>
+      </note>
       </listitem>
      </varlistentry>
      
      <varlistentry>
       <term>Port</term>
        <listitem>
         <para>
+         <!-- What's the default here? -->
```

Comment:
@danielorf What is the default combination for the configuration?

Reply:
Port 636 is default in this implementation.
A little more background:  Port 636 is associated with SimpleTLS and port 389 for StartTLS.  The UI doesn't automatically update the port number based StartTLS status, though it would be a useful feature.